### PR TITLE
Improve getTimezoneOffset description

### LINKED ISSUE #3273: Issue with "Date() constructor": Timezones and DST
MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date

#### What information was incorrect, unhelpful, or incomplete?
Consideration of daylight savings time when inferring local time for date construction.

#### Specific section or headline?
Parameters
- Individual date and time components

#### What did you expect to see?
In addition to the local time inference, it would be nice to also mention that/how DST ist considered.

#### Did you test this? If so, how?
Tested only in Chrome and Node, current timezone is Berlin, Germany (UTC+0100).
```javascript
new Date('2021-03-28').getTimezoneOffset() -> -60
new Date('2021-03-29').getTimezoneOffset() -> -120
```
The created date is not in the timezone I'm _currently_ in, but in the timezone I would be in on the created date (given no geographic change).

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/javascript/reference/global_objects/date/date`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/date/date/index.html
* Last commit: https://github.com/mdn/content/commit/48fca0c614add538e95f1075b2777fc396023d31
* Document last modified: 2021-03-16T07:50:01.000Z

</details>

### LINKED ISSUE #7302: "Date.prototype.getTimezoneOffset()" described as non-deterministic
MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTimezoneOffset

#### What information was incorrect, unhelpful, or incomplete?

The section [Varied results in Daylight Saving Time (DST) time zones](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTimezoneOffset#varied_results_in_daylight_saving_time_dst_time_zones)
 is very confusing, and seems entirely unnecessary.  It appears to describe how the result of `getTimezoneOffset()` can vary whether "the local time zone is currently in DST".  

This is not correct.  [The spec](https://tc39.es/ecma262/multipage/numbers-and-dates.html#sec-local-time-zone-adjustment) says that it will return either zero, or be based on local time zone rules.  It has no dependency on the current date/time.

Also, that section seems to be claiming, without support, that the difference between daylight savings and standard time is "± 60" minutes?

I think the note below it attempts to mitigate the section's inaccuracies by calling it "a simplification".  I don't think it's any simpler than the actual facts.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/javascript/reference/global_objects/date/gettimezoneoffset`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTimezoneOffset
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/date/gettimezoneoffset/index.md
* Last commit: https://github.com/mdn/content/commit/cf607d68522cd35ee7670782d3ee3a361eaef2e4
* Document last modified: 2021-07-20T18:47:57.000Z

</details>

### PR DESCRIPTION
**Pull request by Josh-Cena**
#### Summary

Fix #7302 
Fix #3273

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/date/gettimezoneoffset/index.md [hunk 1]
```diff
@@ -23,60 +23,64 @@ getTimezoneOffset()
...
+| ----------------- | ------------ |
+| UTC-8             | 480          |
+| UTC               | 0            |
+| UTC+3             | -180         |
 
-### Varied results in Daylight Saving Time (DST) time zones
+### Varied results in Daylight Saving Time (DST) regions
 
-In a time zone that annually shifts in and out of Daylight Saving Time (DST), the number of minutes returned by calling `getTimezoneOffset()` can vary.
+In a region that annually shifts in and out of Daylight Saving Time (DST), the number of minutes returned by calling `getTimezoneOffset()` can be non-uniform as `date` varies.
```
Comment (sideshowbarker) [inline files/en-us/web/javascript/reference/global_objects/date/gettimezoneoffset/index.md:44]:
```suggestion
In a region that annually shifts in and out of Daylight Saving Time (DST), the number of minutes returned by calling `getTimezoneOffset()` can be non-uniform as, `date` varies.
```

**Review by sideshowbarker — APPROVED** [review]

File: files/en-us/web/javascript/reference/global_objects/date/gettimezoneoffset/index.md (hunk 1 above)
Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/global_objects/date/gettimezoneoffset/index.md]: Wait, @sideshowbarker doesn't that make it grammatically incorrect?

Comment (sideshowbarker) [reply files/en-us/web/javascript/reference/global_objects/date/gettimezoneoffset/index.md]: > Wait, @sideshowbarker doesn't that make it grammatically incorrect?

oops, yeah — sorry, I put the comma in the wrong place

You want to PR the fix, or you want me to?

Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/global_objects/date/gettimezoneoffset/index.md]: You can go ahead and fix it 👍

**Pull request merged**